framework/db/Query.php

```diff
@@ -553,6 +553,8 @@ public function andWhere($condition, $params = [])
     {
         if ($this->where === null) {
             $this->where = $condition;
+        } elseif (is_array($this->where) && isset($this->where[0]) && strtoupper($this->where[0]) == 'AND') {
```

Comment:
@samdark Good point, though maybe `strcasecmp()` would be better? `strncasecmp()` would also return `0` if `where[0]` was `'andfoobar'`